package/launch-simple-longhorn

```diff
@@ -28,11 +28,10 @@ then
         frontend="tgt-blockdev"
 fi
 
-launcher="longhorn controller"
-extra_opts=""
+longhorn-instance-manager deamon &
```

Comment:
This is misspelled (it's supposed to be `daemon`, and I suspect it's one of the reasons why `launch-simple-longhorn` doesn't work.